# These objects (0 <= status <= 99)
# will get an orange colored visualization marker.

string CATEGORY = UNPICKABLE

### Enums:

uint16 CENTER_OUTSIDE_ROI = 0

uint16 INVALID_BORDER_DISTANCE = 10

uint16 COLLISION_WITH_BOX = 20

uint16 COLLISION_WITH_POINT_CLOUD = 30

uint16 INVALID_Z_AXIS_ORIENTATION = 40

uint16 COLLISION_WITH_BOX_IN_BETWEEN_OBJECT_AND_ROI = 50
